src/gstreamer/gstkvssink.cpp

```diff
@@ -249,6 +249,12 @@ void kinesis_video_producer_init(GstKvsSink *kvssink)
     string region_str;
     bool credential_is_static = true;
 
+    // This needs to happen after we've read in ALL of the properties
+    if (kvssink->disable_buffer_clipping == FALSE) {
```

Comment:
Don’t compare Boolean against literal. Do if (!blah) {}

Reply:
I really didn't see the reason to do this but you're right, for other people's benefit:
https://developer.gnome.org/glib/unstable/glib-Basic-Types.html#gboolean